pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveDataFragmenter.java

```diff
@@ -20,11 +20,7 @@
  */
 
 import java.io.ByteArrayOutputStream;
-import java.util.List;
-import java.util.ListIterator;
-import java.util.Properties;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
Can you revert back to importing each individual library. This imports an unnecessary amount of libraries.